docs/run/run-jobs-batch.mdx

```diff
@@ -8,30 +8,36 @@ description: How to run quantum computing jobs in batch mode using Qiskit Runtim
 
 Batch mode can shorten processing time if all jobs can be provided at the outset. If you want to submit iterative jobs, use [sessions](sessions) instead.  Using batch mode has these benefits:
 
--  The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially.  
--  There is minimal delay between jobs, which can help avoid drift. 
+-  The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially.
+-  There is minimal delay between jobs, which can help avoid drift.
```

Comment:
Adding "usually" because the delay can be significant if you get unlucky and interleave with large jobs from other users. 

```suggestion
-  There is usually minimal delay between jobs, which can help avoid drift.
```